plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPvlanSetupCommandWrapper.java

```diff
@@ -87,7 +88,7 @@ public Answer execute(final PvlanSetupCommand command, final LibvirtComputingRes
             } else if (command.getType() == PvlanSetupCommand.Type.VM) {
                 final String ovsPvlanVmPath = libvirtComputingResource.getOvsPvlanVmPath();
 
-                final Script script = new Script(ovsPvlanVmPath, timeout, s_logger);
+                final Script script = new Script(ovsPvlanVmPath, timeout.getMillis(), s_logger);
```

Comment:
@abhinandanprateek the `Script` class is in the `com.cloud.utils.script` which is part of the project.  What prevents us from adding a constructor that class?